### PR TITLE
Add some people to rust-lang/rust rotation.

### PR DESCRIPTION
**Pull request by ehuss**
Per the conversation at https://internals.rust-lang.org/t/highfive-rotation/11865, some people have volunteered to help with reviews.

### CODE REVIEW

**Comment by ehuss** [comment]
Note: I'm a little uncertain about adding cuviper since they aren't currently on rotation for anything, and I don't want to make assumptions for whatever team is responsible for llvm (T-compiler?). cuviper has bors rights via the release team.

I'm only moderately certain things will work correctly for the submodules. I looked through the code and did some simple tests, and it seems like it should work.

cc @Mark-Simulacrum @Centril @joshtriplett @cuviper @petrochenkov

**Comment by cuviper** [comment]
> Note: I'm a little uncertain about adding cuviper since they aren't currently on rotation for anything, and I don't want to make assumptions for whatever team is responsible for llvm (T-compiler?). cuviper has bors rights via the release team.

FWIW, I also have push access to rust-lang/llvm-project already. It would probably be a good idea to add others for LLVM reviews too, if I may volunteer them: @hanna-kruppe, @nagisa, @nikic

File: highfive/configs/rust-lang/rust.json [hunk 1]
```diff
@@ -8,34 +8,56 @@
...
-        "appveyor.yml":         ["infra-ci"]
+        "Cargo.lock":               ["@Mark-Simulacrum"],
+        "Cargo.toml":               ["@Mark-Simulacrum"],
+        "src/bootstrap":            ["@Mark-Simulacrum"],
+        "src/build_helper":         ["@Mark-Simulacrum"],
+        "src/doc":                  ["doc"],
+        "src/etc":                  ["@Mark-Simulacrum"],
+        "src/liballoc":             ["libs"],
+        "src/libarena":             ["libs"],
+        "src/libbacktrace":         [],
```
Comment (petrochenkov) [inline highfive/configs/rust-lang/rust.json]: The directory list looks seriously outdated.
Perhaps it needs to be regenerated.

Comment (ehuss) [reply highfive/configs/rust-lang/rust.json]: Yea, I had removed them, but decided this PR was pretty long and harder to review.  Submitted separately as #251.

**Comment by Mark-Simulacrum** [comment]
This looks good to me, I'm happy to merge as-is I think -- do you want to merge #251 and then rebase or vice versa?

**Comment by ehuss** [comment]
Doesn't matter to me.  This first, then 251 might be easier to rebase.

**Pull request merged**